lib/Conversion/TorchToLinalg/Uncategorized.cpp

```diff
@@ -2876,9 +2896,11 @@ class ConvertInterpolateOp
                 [&](OpBuilder &b, Location loc, ValueRange args) {
                   Value retVal;
                   if (mode.substr(0, 7) == "nearest") {
+                    std::string coordTfMode = mode.substr(7, mode.find(","));
+                    std::string nearestMode = mode.substr(mode.find(",") + 1);
```

Comment:
If this op was coming from native torch, it would not have a comma in the mode string. Would this cause an error?

Reply:
I think you should change it to only have a comma if the onnx.Resize op has a nearest_mode besides floor, and then have the nearestMode defined here be `""` or the substring, conditionally on whether the comma is found.